packages/config/src/VitessceAutoConfig.js

```diff
@@ -362,13 +527,10 @@ async function generateConfig(url, vc) {
     return Promise.reject(error);
   }
 
-  const dataset = vc
-    .addDataset(configInstance.fileName)
-    .addFile(fileConfig);
+  dataset.addFile(fileConfig);
```

Comment:
I am now creating only 1 dataset and all files are added as part of the same dataset. Before, for every file I would create a new dataset.